### PR TITLE
update sa and namespace used in kustomize

### PR DESCRIPTION
**Pull request by horis233**
**What this PR does / why we need it**:

1. Update serviceAccountName in the CSV
2. Replace the namespace in the kustomization

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: config/certmanager/certificate.yaml [hunk 1]
```diff
@@ -6,15 +6,15 @@ apiVersion: cert-manager.io/v1alpha2
 kind: Issuer
 metadata:
   name: selfsigned-issuer
-  namespace: system
+  namespace: ibm-common-services
```
Comment (DanielXLee) [inline config/certmanager/certificate.yaml:9]: ODLM don't only for ibm-common-servcies, so don't hard code this namespace, keep `system` namespace

Comment (horis233) [reply config/certmanager/certificate.yaml:9]: OK, sure

Comment (horis233) [reply config/certmanager/certificate.yaml]: @DanielXLee I revert the change to use back the `namespace: system`

**Comment by horis233** [comment]
@chenzhiwei @DanielXLee @DanielXLee This PR is ready to be merged.

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**